languages/csharp/exercises/concept/chars/.meta/Example.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Text;
+
+public static class Identifier
+{
+    public static string Clean(string str)
```

Comment:
Maybe use `identifier` as the parameter name?

```suggestion
    public static string Clean(string identifier)
```

Reply:
The implementation looks really nice 👍